ts/components/cie/CieRequestAuthenticationOverlay.tsx

```diff
@@ -308,12 +308,19 @@ const ErrorComponent = (
   props: { onRetry: () => void } & Pick<Props, "onClose">
 ) => (
   <SafeAreaView style={[IOStyles.flex, styles.errorContainer]}>
```

Comment:
Do we need two `SafeAreaView`, one outside and one inside the `OperationResultScreenContent`? Note that both are exported from `react-native` thus they only work on iOS.

Reply:
Good catch!

Reply:
Fixed here bdef5749ed4875421f00232343e2f3201ea6fc8c